app/models/neighborhood.rb

```diff
@@ -88,35 +81,17 @@ class Neighborhood
       longitude: "-122.4900",
       name: "The Sunset"
     },
-    unionsquare: {
+    union_square: {
       latitude: "37.7881",
       longitude: "-122.4075",
       name: "Union Square"
     },
   }
 
-  def self.city(key)
-    if @@cities[key].present?
-      OpenStruct.new(@@cities[key])
-    else
-      nil
-    end
-  end
-
-  def self.city_latlng(key)
-    city = Neighborhood.city(key)
-
-    if city.present?
-      LatLng.new(city.latitude, city.longitude)
-    else
-      nil
-    end
-  end
-
   def self.districts
     unless @districts
       @districts = []
-      @@districts.each do |key, district|
+      DISTRICTS.each do |key, district|
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.